Microsoft.Toolkit.Uwp.SampleApp/SamplePages/Microsoft Graph Service/MicrosoftGraphPage.xaml.cs

```diff
@@ -117,7 +117,9 @@ public MicrosoftGraphPage()
                 top = Convert.ToInt32(txtTop);
             }
 
+            var source = new MicrosoftGraphSource();
```

Comment:
Putting to code in the second allows for null to be passed to the constructor. This should be an ArgumentNullException for the ctor. Allow for backward compatibility with the current ctor and enforce proper arguments for the new ctor